Formula/taskell.rb

```diff
@@ -0,0 +1,25 @@
+require "language/haskell"
+
+class Taskell < Formula
+  include Language::Haskell::Cabal
+
+  desc "Command-line Kanban board/task manager with support for Trello"
+  homepage "https://taskell.app"
+  url "https://github.com/smallhadroncollider/taskell/archive/1.2.5.tar.gz"
+  sha256 "818055fcfe0656b3bcf00c37fdf2ddd5d2958bb169f5eda571bf4957e7aa64d8"
+
+  depends_on "cabal-install" => :build
+  depends_on "ghc" => :build
+
+  def install
+    cabal_sandbox do
+      cabal_install "hpack"
+      system "./.cabal-sandbox/bin/hpack"
+      install_cabal_package
+    end
+  end
+
+  test do
+    system "#{bin}/taskell", "-v"
```

Comment:
We need a test that exercises the some of the functionality of the app.

Reply:
I'll need to add some functionality to taskell so that it outputs something testable to stdout.